src/java_bytecode/java_utils.cpp

```diff
@@ -20,3 +22,94 @@ bool java_is_array_type(const typet &type)
     to_struct_type(type).get_tag()),
                     "java::array[");
 }
+
+unsigned java_parameter_count(
```

Comment:
@smowton @mgudemann, I can reimplement the function by calling indeed `java_type_from_string` and then use `count_slots` (L111) in an iteration through the arguments. Should I move the resulting reimplementation to `java_types.{h,cpp}` or I still leave it in `java_utils.{h,cpp}`?